composer.json

```diff
@@ -11,12 +11,12 @@
         "ext-json": "^1.6",
         "ext-PDO": "^7.2",
         "doctrine/dbal": "^2.9",
-        "google/protobuf": "^3.7",
         "google/cloud-spanner": "^1.15",
-        "grpc/grpc": "^1.19"
+        "phpmyadmin/sql-parser": "dev-master"
     },
     "require-dev":  {
-        "phpunit/phpunit": "^8.1"
+        "phpunit/phpunit": "^8.1",
```

Comment:
Wouldn't it conflict with a https://github.com/oat-sa/generis/pull/735/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R75 ?

Reply:
we can leave it there as this does not affect the root composer usage